src/wallet/test/CMakeLists.txt

```diff
@@ -13,9 +13,7 @@ target_sources(test_bitcoin
     feebumper_tests.cpp
     group_outputs_tests.cpp
     init_tests.cpp
-    ismine_tests.cpp
```

Comment:
In dc7bf5fd6a320c4528a28cef2a565366bbab3877 "wallet: Delete LegacySPKM": this test (file) and the one below still exist.

Reply:
Apparently since this PR was opened, descriptor specific tests were added to those files. I've removed the legacy ones from them and added back to the `CMakeLists.txt`.